tuxemon/core/monster.py

```diff
@@ -432,6 +437,48 @@ def get_evolution(self, path):
                     return evolution['monster_slug']
         return None
 
+    def get_sprite(self, sprite, **kwargs):
+        """Gets a specific type of sprite for the monster.
+
+        :rtype: Pygame surface
+        :returns: The surface of the monster sprite
+        """
+        if sprite == "front":
+            surface = tools.load_sprite(self.front_battle_sprite, **kwargs)
+        elif sprite == "back":
+            surface = tools.load_sprite(self.back_battle_sprite, **kwargs)
+        elif sprite == "menu":
+            surface = tools.load_sprite(self.menu_sprite, **kwargs)
+        else:
+            raise ValueError("Cannot find sprite for: {}".format(sprite))
+
+        # Apply flairs to the monster sprite
+        for flair in self.flairs:
+            flair_path = self.get_sprite_path("gfx/sprites/battle/" + self.slug + "-" + sprite + "-" + flair['value'])
+            if flair_path != MISSING_IMAGE:
+                flair_sprite = tools.load_sprite(flair_path, **kwargs)
```

Comment:
if we dont crash and also cannot find the flair, please add a logging statement here so it can be debugged